frontend/src/pages/GraphPF/GraphPF.tsx

```diff
@@ -886,6 +887,10 @@ const TopologyContent: React.FC<{
                   zoomOutCallback: () => {
                     controller && controller.getGraph().scaleBy(ZOOM_OUT);
                   },
+                  // currently unused
+                  fitToScreenCallback: () => {
+                    controller.getGraph().fit(80);
```

Comment:
Why do you hardcode 80 here? It should be the FIT_PADDING value right?

Reply:
good grief, I need a holiday, it seems.  That code isn't executed, but yes.  I'll push the change.

Reply:
done.